Cookbook/Technical-Documents/ToolsAndServices.md

```diff
@@ -17,22 +17,21 @@
 | [CircleCI](https://circleci.com/gh/Babylonpartners) | See CI jobs and logs | Use your GitHub login |
 | [Pull Panda](https://pullreminders.com) | Stats on PRs and reminders for when you've been asked for a review | Use your GitHub login |
 | [HockeyApp](https://rink.hockeyapp.net/) | see OTA builds sent internally to QA | Ask support engineer in the iOS team channel on Slack to invite you to the org |
-| [Firebase](https://console.firebase.google.com/u/1/) | View crash reports, performance monitoring | Ask Rui or David for access |
+| [Firebase](https://console.firebase.google.com/u/1/) | View crash reports, performance monitoring | Ask Catarina or David for access |
 
 ## Internal tools
 
 * Xcode templates
 
   These templates are helpful when implementing new features or adding tests for existing ones. To install them run `Templates/install_xcode_templates.sh`
-  
+
 * Git hooks
 
   If you want you can install additional git hooks which will check for pods and Xcode templates to be in sync when switching branches or pulling from remote. To install them follow instructions in `githooks/README.md`
-  
+
 * Slack bots
-  
-  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.
-  
+
+  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands. Check [this guide](TBD) for details.  
```

Comment:
If there is no guide let's remove this step.

```suggestion
  You can trigger CI jobs from Slack in the `ios-build` channel using slash commands.
```

Reply:
Or maybe @ilyapuchka has written one

Reply:
yep [here](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/SlackCIIntegration.md). I'll add that